folium/plugins/minimap.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import (absolute_import, division, print_function)
+
+import json
+
+from branca.element import CssLink, Figure, JavascriptLink, MacroElement
+from folium.raster_layers import TileLayer
+
+from jinja2 import Template
+
+
+class MiniMap(MacroElement):
+    """Add a minimap (locator) to an existing map.
+
+    Uses the Leaflet plugin by Norkart under BSD 2-Clause "Simplified" License.
+    https://github.com/Norkart/Leaflet-MiniMap
+
+    Parameters
+    ----------
+    tile_layer: folium tilelayer, str (Optional)
+
```

Comment:
No need to have empty lines between the parameters

Reply:
fixed

Reply:
I'm still seeing them on the 'files changed' page, am I missing something?